tests/rust/tcp-test/bind/mod.rs

```diff
@@ -245,7 +245,7 @@ fn bind_to_non_local_address(libos: &mut LibOS) -> Result<()> {
 
     // Fail to bind socket.
     match libos.bind(sockqd, addr) {
-        Err(e) if e.errno == libc::EADDRNOTAVAIL => (),
+        Err(e) if e.errno == libc::EADDRNOTAVAIL || e.errno == libc::EBADF => (),
```

Comment:
Where is this bug comming from?

Reply:
socket2 returns EBADF instead from this call.

Reply:
Sure, but then this does not comply with expected behavior for sockets. Shouldn't we thus interpose the returned error code?

Reply:
Oh nooo

Reply:
I'm not sure how to distinguish it from the socket being closed if I just translate all EBADF to EADDRNOTAVAIL

Reply:
Let me revisit this if/when we move to TcpStream